elks/arch/i86/drivers/char/Makefile

```diff
@@ -63,6 +63,9 @@ else
 ifdef CONFIG_CONSOLE_DIRECT
 OBJS += console-direct.o bell-8254.o
 endif
+ifdef CONFIG_CONSOLE_DUAL
+OBJS += crtc-6845.o
+endif
```

Comment:
Lets put this ifdef inside the CONFIG_CONSOLE_DIRECT above, so there's no chance it won't cause a kernel compilation failure if the .config file gets a bit out of sync.

Reply:
Fixed. Not sure why Github doesn't always show the `outdated` tag on these when I push changes.